### PR TITLE
Add UL bbH_ybyt cards

### PR DESCRIPTION
**Pull request by adewit**
This adds the cards for the bbH_ybyt (interference term) process. A gridpack was produced following the instructions here: https://github.com/panwarlsweet/bbH/tree/MG5_v2.6_UL19 - these are the resulting proc, run, and param cards for completeness. The precompiled process directories from the repository are needed to produce CMS gridpacks. 

The gridpack is available at: /eos/cms/store/user/adewit/bbH_4FS_ybyt_M-125_tarball.tar.xz , if the cards are OK, can this gridpack be moved to cvmfs so that it can be used for central MC requests? If a different procedure is required for that please let me know. 

Of course, if changes to the cards are needed I'll reproduce the gridpack.

### CODE REVIEW

**Comment by adewit** [comment]
Just a ping to hopefully avoid this one getting lost (PR made just before the CERN closure). Let me know if there are any comments!

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/bbH_ybyt/bbH_ybyt_proc_card.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/bbH_ybyt/bbH_ybyt_proc_card.dat:15]: please remove the unnecessary lines (from 'import model sm' to 'define vl~')

File: bin/MadGraph5_aMCatNLO/cards/production/13TeV/bbH_ybyt/bbH_ybyt_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+ 100 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+ 10000 = nevt_job! Max number of events per job in event generation.
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/13TeV/bbH_ybyt/bbH_ybyt_run_card.dat:33]: Please change this to -1

**Comment by sihyunjeon** [comment]
The changes suggested above won't really make you to reproduce the gridpacks. It's a suggestion to make the genproductions neat

**Comment by sihyunjeon** [comment]
Oops the accuracy should stay as it was. So I changed it back.
@agrohsje I am fine with merging this. (maybe you want to change the pdf set?)

**Comment by agrohsje** [comment]
Hi @adewit . Sorry for the delay. I took 3 weeks off and ignored mails. Cards look good. I just did some cosmetic changes that have no impact. PDF set will be the same but used a template string to make it future-compatible.
Maybe one question: Do you know why internal madloop dependencies are used?

**Comment by adewit** [comment]
@agrohsje Hmm, I don't know exactly why internal madloop dependencies are used. As mentioned in the PR description, these cards are based on a precompiled process directory (from the authors of the process) - all I know is that these dependencies are also used in the "main" bbH process (with ybyb only), which follows the same generation procedure. If it would be useful I can try to investigate more.

**Comment by agrohsje** [comment]
@adewit don't worry. that's fine. @sihyunjeon can you check my small changes and merge. just to be sure i didn't screw anything with my online edits.

**Comment by adewit** [comment]
@agrohsje ok, perfect. BTW, once this is merged, how should I go about getting the gridpack mirrored on cvmfs? Do I need to ask the relevant MC contact to do this?

**Comment by sihyunjeon** [comment]
@adewit yes your pag mc contacts will have rights to push things to cvmfs

**Comment by sihyunjeon** [comment]
merging!

**Pull request merged**